e2e/tests/visual/planning.visual.spec.js

```diff
@@ -24,29 +24,35 @@ const { test } = require('../../pluginFixtures');
 const { setBoundsToSpanAllActivities } = require('../../helper/planningUtils');
 const { createDomainObjectWithDefaults, createPlanFromJSON } = require('../../appActions');
 const percySnapshot = require('@percy/playwright');
-const examplePlanLarge = require('../../test-data/examplePlans/ExamplePlan_Large.json');
+const examplePlanSmall = require('../../test-data/examplePlans/ExamplePlan_Small2.json');
+
+const snapshotScope = '.c-object-view';
 
 test.describe('Visual - Planning', () => {
     test.beforeEach(async ({ page }) => {
         await page.goto('./', { waitUntil: 'networkidle' });
     });
     test('Plan View', async ({ page, theme }) => {
         const plan = await createPlanFromJSON(page, {
-            json: examplePlanLarge
+            json: examplePlanSmall
         });
 
-        await setBoundsToSpanAllActivities(page, examplePlanLarge, plan.url);
-        await percySnapshot(page, `Plan View (theme: ${theme})`);
+        await setBoundsToSpanAllActivities(page, examplePlanSmall, plan.url);
+        await percySnapshot(page, `Plan View (theme: ${theme})`, {
+            scope: snapshotScope
```

Comment:
interesting

Reply:
Yeah! This is the same thing we do with the `tree` visual tests. Here we are only snapping the object view, so only the parts relevant to the Plan/Gantt Chart views